newIDE/app/src/UI/KeyboardShortcuts/index.js

```diff
@@ -68,24 +86,81 @@ export default class KeyboardShortcuts {
     return this._shiftPressed;
   }
 
+  shouldFollowAxis() {
+    return this._shiftPressed;
+  }
+
+  shouldNotSnapToGrid() {
+    return this._altPressed;
+  }
+
+  shouldResizeProportionally() {
+    return this._shiftPressed;
+  }
+
+  shouldScrollHorizontally() {
+    return this._altPressed;
+  }
+
+  shouldMoveView() {
+    return this._spacePressed || this._mouseMidButtonPressed;
+  }
+
+  shouldZoom() {
+    if (isMacLike()) {
+      return this._isControlOrCmdPressed();
+    } else {
+      if (
+        !this._isControlOrCmdPressed() &&
```

Comment:
nit: this can just be `return !this._isControlOrCmdPressed() && !this._altPressed && !this._shiftPressed`